main/lsp/test/error_reporter_test.cc

```diff
@@ -51,17 +55,22 @@ TEST_CASE("NotifiesVSCodeWhenFileHasErrors") {
     auto gs = makeGS();
     auto cs = makeConfig();
     ErrorReporter er(cs);
-    vector<unique_ptr<core::Error>> errors;
     {
         core::UnfreezeFileTable fileTableAccess(*gs);
         auto epoch = 0;
         auto file = make_shared<core::File>("foo.rb", "foo", core::File::Type::Normal, epoch);
         auto fref = gs->enterFile(file);
         auto outputVector = dynamic_pointer_cast<LSPOutputToVector>(cs->output);
 
+        vector<unique_ptr<core::Error>> errors;
         errors.emplace_back(
             make_unique<core::Error>(core::Loc(fref, 0, 0), core::ErrorClass{1, core::StrictLevel::True}, "MyError",
                                      vector<core::ErrorSection>(), vector<core::AutocorrectSuggestion>(), false));
+
+        vector<unique_ptr<Timer>> diagnosticLatencyTimers;
+        diagnosticLatencyTimers.emplace_back(make_unique<Timer>(logger, "last_diagnostic_latency"));
```

Comment:
It doesn't look like this test is using these timers. Is there a benefit to creating them vs. passing an empty list?